cmd/ftl/cmd_new.go

```diff
@@ -84,8 +93,59 @@ func (i newGoCmd) Run(ctx context.Context) error {
 	return nil
 }
 
+func (i newJavaCmd) Run(ctx context.Context) error {
+	return RunJvmScaffolding(ctx, i.Dir, i.Name, i.Group, java.Files())
+}
+
 func (i newKotlinCmd) Run(ctx context.Context) error {
-	return fmt.Errorf("kotlin scaffolinging temporarily removed")
+	return RunJvmScaffolding(ctx, i.Dir, i.Name, i.Group, kotlin.Files())
+}
+
+func RunJvmScaffolding(ctx context.Context, dir string, name string, group string, source *zip.Reader) error {
+	name, path, err := validateModule(dir, name)
+	if err != nil {
+		return err
+	}
+
+	config, err := projectconfig.Load(ctx, "")
```

Comment:
This needs to be passed in, it's created from `--config` in main and so is injectable to `Run()` methods.